tests/importer/model_flattening.cpp

```diff
@@ -28,6 +28,8 @@ TEST(ModelFlattening, invalidInput)
 
     EXPECT_EQ(nullptr, flatModel);
     EXPECT_EQ(size_t(1), importer->issueCount());
+    EXPECT_EQ("The given model is invalid.", importer->issue(0)->description());
```

Comment:
Shouldn't the expected issue description be defined as a `const std::string`?

Reply:
No, doing it this way works just as well.

Reply:
I meant that all our tests follow the following template:
```c++
const std::string error = "Some error description.";

...

EXPECT_EQIerror, something->error());
```

So, I am merely asking for consistency, so that it makes the code easier to read and maintain.

Reply:
I don't think we want strict consistency in the tests, in the library yes, tests no.